lib/galaxy/managers/taggable.py

```diff
@@ -122,9 +122,9 @@ def _create_tag_filter(model_class=None):
                 if ":" not in val:
                     # We require an exact match and the tag to look for has no user_value,
                     # so we can't just concatenate user_tname, ':' and user_vale
-                    cond = target_model.table.c.user_tname == val
+                    cond = target_model.table.c.user_tname.ilike(val)
```

Comment:
Do we need to escape `%` and `_` ? Should we instead cast to lowercase ?

```python
from sqlalchemy import func
```

```suggestion
                    cond = func.lower(target_model.table.c.user_tname) == val.lower()
```
I've not tested this, but it seems like a good idea to keep it simple(r) ?

Reply:
Tested, and it works as well, thanks!

Reply:
+1, we use `lower` in the model for comparison I believe